macros/edr/tests/test_utils/create_elementary_test_table.sql

```diff
@@ -0,0 +1,21 @@
+{% macro create_elementary_test_table(database_name, schema_name, test_name, table_type, sql_query) %}
+    {% if execute %}
+        {% set temp_table_name = elementary.table_name_with_suffix(test_name, "__" ~ table_type) %}
+        {{ elementary.debug_log(table_type ~ ' table: ' ~ database_name ~ '.' ~ schema_name ~ '.' ~ temp_table_name) }}
+
+        {% set _, temp_table_relation = dbt.get_or_create_relation(database=database_name,
+                                                                   schema=schema_name,
+                                                                   identifier=temp_table_name,
+                                                                   type='table') -%}
+
+        {# Save the test table to the graph for easy access later #}
+        {% do graph.setdefault("elementary_test_tables", {}) %}
```

Comment:
In another PR we created a root `elementary` key on the graph so maybe we should store everything that we do there like `graph["elementary"]["test_tables"]`.